BedrockCommand.cpp

```diff
@@ -207,29 +207,3 @@ void BedrockCommand::finalizeTimingInfo() {
         }
     }
 }
-
-// pop and push specializations for SSynchronizedQueue that record timing info.
-template<>
-unique_ptr<BedrockCommand> SSynchronizedQueue<unique_ptr<BedrockCommand>>::pop() {
```

Comment:
Why don't we need these anymore? Do we *always* use `BedrockTimeoutCommandQueue` now?

Reply:
As far as I can tell these are totally unused. I looked for the log line in `push` `Enqueueing command` and I can't find any occurrences in the last 7 days.

Reply:
If you can delete them and it still compiles, it means we're falling back to some other function that works in their place, so there's *some* `push` and `pop` that function here. If it's in `BedrockTimeoutCommandQueue` then I think it works fine without these. If it's in the base `SSynchronizedQueue`, then it seems like this would break some time counting, though it seems like it doesn't so I guess we can go ahead with it.